pkgs/by-name/em/empty-epsilon/package.nix

```diff
@@ -14,7 +14,22 @@
 }:
 
 let
-  version = {
+  glm' = glm.overrideAttrs (
```

Comment:
https://github.com/NixOS/nixpkgs/blob/2bc291770f4ac88224e6c211d341899a00fee472/pkgs/README.md#overrideattrs-and-overridepythonattrs

Reply:
But glm is not using the Python builder 😅
https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/gl/glm/package.nix#L8

We are probably looking at different things or it is more complex than on first sight and then I will probably change it to be more obvious

Reply:
I am confused about the confusion here :) The linked documentation doesn't say anything about the Python builder other than grouping `.overridePythonAttrs` in with `.overrideAttrs`. The problem is the `.overrideAttrs`. (If the wording makes it sound like it's only about Python packages we should try to fix that - IIRC https://github.com/NixOS/nixpkgs/pull/421201 started with just `.overrideAttrs` but then it was brought up that `.overridePythonAttrs` is similar.)

Reply:
I didn't help that when opening the link in a new tab I end up at the sources headline 😅